frontend/src/contexts/HostContext/hostReducerMiddleware.js

```diff
@@ -207,6 +207,21 @@ function czarSelectWinner({ players, playerIDs }) {
   });
```

Comment:
I would test that it sends the expected message when it receives the appropriate action type.